src/features/Consumer/components/StatDisplay.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+  Group,
+  Paper,
+  Text,
+  ThemeIcon,
+  SimpleGrid,
+  Container,
+  Box,
+} from "@mantine/core";
+import { IconArrowUpRight, IconArrowDownRight } from "@tabler/icons-react";
+
+const data = [
+  { title: "Weighted No of Stars", value: "4.0 Stars", diff: -20 },
+  { title: "No. of Real Reviews", value: "420", diff: -69 },
+];
+
+export const StatDisplay: React.FC = () => {
+  const stats = data.map((stat) => {
+    const DiffIcon = stat.diff > 0 ? IconArrowUpRight : IconArrowDownRight;
+
+    return (
+      <Paper
+        withBorder
+        p="xl"
+        radius="md"
+        key={stat.title}
+        style={{ height: "100%" }}
+      >
+        <Group justify="apart" style={{ height: "100%" }}>
+          <div>
+            <Text
+              c="dimmed"
+              tt="uppercase"
+              fw={700}
+              fz="lg"
+            >
+              {stat.title}
+            </Text>
+            <Text fw={700} fz="xl">
+              {stat.value}
+            </Text>
+            <Text c="dimmed" fz="xl" mt="xl">
+              <Text
+                component="span"
+                c={stat.diff > 0 ? "teal" : "red"}
+                fw={700}
+              >
+                {stat.diff}%
+              </Text>{" "}
+              {stat.diff > 0 ? "More" : "Less"} than Google Reviews
+            </Text>
+          </div>
+          <ThemeIcon
+            color="gray"
+            variant="light"
+            style={{
+              color:
+                stat.diff > 0
+                  ? "var(--mantine-color-teal-6)"
+                  : "var(--mantine-color-red-6)",
+            }}
```

Comment:
The `c` prop corresponds to the `color` style property and because it's a mantine component you can just do teal.6 or red.6 also!
```suggestion
            c={stat.diff > 0 ? "teal.6" : "red.6"}
```